src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -83,7 +85,41 @@ func init() {
 	rand.Seed(time.Now().UnixNano())
 }
 
+func logPanic(msg string) {
+	cfg, err := config.Init(packageVersion)
+	if err != nil {
+		panic("Error initializing config")
+	}
+	if err := logging.Init(); err != nil {
+		panic("Error initializing logging")
+	}
+
+	<-logging.Configure("", "", cfg.InstanceId, version, revisionDate)
+
+	log.Error(msg)
```

Comment:
Actually what will that ```msg``` here typically end up being? I'm wondering if we should prepend it with something we can easily search like "PANIC" or something. We should definitely add panic searches to support.